src/moduler/filtrering/filter/filter-utils.ts

```diff
@@ -23,6 +23,7 @@ export interface Periode {
 
 export function selectDatoErIPeriode(dato: string, state: any): boolean {
     const historiskPeriode = selectHistoriskPeriode(state);
+    // @ts-ignore : for å unngå runtimefeil "Argument of type 'any' is not assignable to parameter of type 'never'".
```

Comment:
denne må vi finne ut av.

Reply:
En mulig av er utfunnet ved at jeg deklarerer type eksplisitt på metoden som filtrerer ut oppfølgingsperioder som har sluttdato. Måtte flytte den over til en ts-fil for å kunne gjøre det. Jeg er på litt dypt vann så best å sjekke om du er enig..